config/smarty.config.inc.php

```diff
@@ -113,6 +113,7 @@ function smartyEscape($string, $esc_type = 'html', $char_set = null, $double_enc
 smartyRegisterFunction($smarty, 'modifier', 'ucfirst', 'ucfirst');
 smartyRegisterFunction($smarty, 'modifier', 'urlencode','urlencode');
 smartyRegisterFunction($smarty, 'modifier', 'htmlspecialchars','htmlspecialchars');
+smartyRegisterFunction($smarty, 'modifier', 'implode', 'implode');
```

Comment:
```suggestion
smartyRegisterFunction($smarty, 'modifier', 'implode', 'implode');
smartyRegisterFunction($smarty, 'modifier', 'explode', 'explode');
smartyRegisterFunction($smarty, 'modifier', 'print_r','print_r');
smartyRegisterFunction($smarty, 'modifier', 'var_dump','var_dump');
smartyRegisterFunction($smarty, 'modifier', 'lcfirst','lcfirst');
smartyRegisterFunction($smarty, 'modifier', 'nl2br','nl2br');
```

Could you add these ones please as well? I wonder if there are other we could add that have chances to be widely used 🤔

Reply:
I see we use `$php_errors|@count` in some templates, but I don't know if it needs this additional register or if it's already handled by Smarty And more globally do all these modifiers work if they are used with a `@` prefix?